cloudinit/writefiles.go

```diff
@@ -44,15 +44,13 @@ type files struct {
 	Append      bool   `json:"append,"`
 }
 
-var _ cloudCongfigAction = &writeFilesAction{}
```

Comment:
I think we should preserve this; it helps to recognize when the implementation diverges from the expected interface

Reply:
because of this line we have the exact same behavior.

https://github.com/kubernetes-sigs/cluster-api-provider-docker/pull/201/files#diff-b6f72e79f2e82238d996fd9a741ab9caR47